config.go

```diff
@@ -198,6 +195,56 @@ func Load() {
 	}
 }
 
+func postgresFromEnv(cfg *Prest) (err error) {
+	// rewrite all pg config
+	if os.Getenv("PREST_PG_URL") != "" {
```

Comment:
I believe it is better if you use the _LookupEnv_ function instead of calling _GetEnv_ twice for each variable.
https://golang.org/pkg/os/#example_LookupEnv

Reply:
Done